models/template_l1_APU-2.json

```diff
@@ -0,0 +1,88 @@
+{
+  "arch": 2,
+  "productURL": "https://pcengines.ch/apu2.htm",
+  "productStatus": "production",
+  "attr": {
+    "memory": "4G",
+    "storage": "56G",
+    "Cpus": "4"
+  },
+  "logo": {
+    "logo_front": "/workspace/spec/logo_front_APU-2.png",
+    "logo_back": "/workspace/spec/logo_back_APU-2.png"
+  },
+  "ioMemberList": [
+    {
+      "ztype": 1,
```

Comment:
Hi, @rvs
Sorry for the late response. As I can see inside EVE (https://github.com/lf-edge/eve/blob/103bd43f758260fb5ed1336917e9441135a91155/api/go/config/devmodel.pb.go#L353) we have the similar structure, but probably we should use `ptype`, not `ztype` here?

Reply:
Now that you asked it -- I'm actually not sure -- these are taken verbatim from zedcloud controller and they all work out of the box there. So...